.github/workflows/freezeyt_blog.yaml

```
@@ -22,12 +22,14 @@ jobs:
     steps:
       # Checks out a copy of your repository on the ubuntu-latest machine
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
           python-version: '3.8'
+          cache: 'pip'
+          cache-dependency-path: '**/setup.cfg'
```

`setup-python`'s default behavior is to look for  requirements.txt in all subdirectories, but here we don't need the `**/`.
We also might want to switch to `pyproject.toml` entirely in the future, [when setuptools supports it](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html), so consider:
```suggestion
          cache-dependency-path: |
              setup.cfg
              pyproject.toml
```

---

Ok we can discuss it on stream because I am not sure about that